test.js

```diff
@@ -1,90 +1,120 @@
 // var log = require('why-is-node-running');
 var tape = require('blue-tape');
 var run = require('./index').runParams;
-var when = require('when');
 var loadtest = require('loadtest');
 
+function lift(fn) {
```

Comment:
why not simply:
```javascript
const promisify = fn =>function(){
    return new Promise(resolve => resolve(fn.apply(this, arguments)));
};
```
Note that your fn.apply changes `this` to `fn`, which I believe is an error.

Also maybe change the name from `lift`, as in original code `when.lift` was an error, as `when.lift` supposed to be used on node callback style functions, while ours were not such.